presto-docs/src/main/sphinx/functions/system.rst

```diff
@@ -0,0 +1,14 @@
+===================
+Session Information
+===================
+
+Functions providing information about the query execution environment.
+
+String Functions
```

Comment:
Remove this header. The name "String" is a bit strange, and it doesn't provide any value since there is only one section in this chapter.